marklogic-data-hub-central/ui/src/config/tooltips.config.js

```diff
@@ -92,13 +92,22 @@ const AdvCustomOtherTooltips = {}
 
 const ModelingTooltips = {
   entityTypeName: 'Click to view or edit settings for this entity type.',
+  addNewEntity: 'Click here to add a new entity type',
```

Comment:
This does not end with a period - which is what I'd vote for, but the other messages all end with a period, so I'm thinking this should end with a period too for consistency.